spec/bundler/plugin_spec.rb

```diff
@@ -80,6 +80,38 @@
     end
   end
 
+  describe "uninstall command" do
+    before do
+      allow(installer).to receive(:uninstall).with(["new-plugin"]) do
```

Comment:
`installer` here is `Plugin::Installer` which is not being used for `uninstall`. So, I think we can skip this.